app/controllers/application_controller.rb

```diff
@@ -1,14 +1,22 @@
 class ApplicationController < ActionController::API
+  rescue_from JWT::DecodeError, with: :error_invalid
 
   def secret_key
-    "drinkthis"
+    ENV['GOOGLE_CLIENT_SECRET']
```

Comment:
Nice -- definitely better to use an ENV variable here.

Reply:
:)